README.md

```diff
@@ -324,6 +324,45 @@ Jekyll::Assets::Hook.register :env, :init do
   # Your Work
 end
 ```
+## Combining Multiple Scripts / Stylesheets
```

Comment:
Can you please add an empty line before the this heading so that it won't be "joined" (visually) with block before.